### ISSUE TITLE
Create new website page for "support Lookit" info

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#854]
# TL;DR
In the experimenter view, add a new link to the header bar that links to a static page describing ways that people using Lookit can help it succeed as a project. 

# Narrative
Create a single place for Lookit researchers to learn about community norms on citation and writing Lookit into grants when possible. 

# Acceptance Criteria

Create an MVP version with the copy below

# Implementation Notes

We'll do this using the new template system, CJ will review structure with Melissa so she can update them (expect to add a bunch of changes to this in the near future, but want to get a simple version up now)

# Copy!

Tab Title: Support Lookit!
Page Title: How Researchers can Support Lookit

Lookit is free for labs to use, but like any open source software project, it's not free to run! On this page, you can learn about ways to keep Lookit running (and getting better!) for a long time to come.

## Cite Lookit!

The biggest way you can support Lookit is by citing us in any paper (or poster, or presentation, etc.) that you publish using data you collected on Lookit, or that depends on the site in some other way.  This helps us demonstrate our impact to funders.

Scott, K., & Schulz, L. (2017). Lookit (part 1): A new online platform for developmental research. Open Mind, 1(1), 4-14. https://doi.org/10.1162/OPMI_a_00002

If you create an experiment that is closely based on one made by another researcher, you should also cite or credit that person's work - reach out to the study creator to learn the best way to do this!

## Tell us how you use Lookit

Developmental psychology is slow, and this can put us at a disadvantage especially when talking about software with people from other fields that get to publication more quickly.  You can help us a TON by letting us know any time you have a milestone. Milestones could include:

* Posting a preprint or submitting a paper
* Presenting a poster or conference talk
* Making guides or teaching materials to help others learn to use Lookit
* Submitting a grant that plans to use Lookit
* A surprising or cool finding
* A new or unexpected way to use Lookit

We would LOVE to showcase your work on the Lookit website - please fill in the form below to tell us about it!

```
<iframe src="https://docs.google.com/forms/d/e/1FAIpQLScy0DBdFNvDPsd-YkqiDkykoUmpjEfZCObOBefW7w-ekw5v0w/viewform?embedded=true" width="640" height="1900" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
```

Fallback if that fails
```
<a href='https://forms.gle/9tLA2stdJZFKP5zz9'>Access the form here!</a>
```

## Write Lookit into your grants

Lookit is free for researchers to use, but if you are seeking funding, we would really appreciate you planning for funds to cover the cost of using the site.  There are two main ways that Lookit might be involved in a funding application:

* As a service, covering the cost of testing N participants during the course of your grant (contact us for ideas about how to calculate and write Lookit testing into your budget)
* As a collaborator, to implement something new that would expand what Lookit can do.

We would be thrilled to discuss any and all proposals you may be working on - contact Melissa (mekline@mit.edu) for any inquiries!

## Our Funders & Contributors

Finally, a big thank-you to all the funders and contributors that have made Lookit possible:

* The National Science Foundation
* DARPA
* MIT Center for Brains, Minds and Machines
* The University of Tokyo
* Halis Family Foundation

**Comment by mekline** [comment]
Copy edits from Laura meeting - missing headings have no content changes

## Cite Lookit!

The biggest way you can support Lookit is by citing us in any paper (or poster, or presentation, etc.) that you publish using data you collected on Lookit, or that depends on the site in some other way. This helps us demonstrate our impact to funders and potential funders!  In your methods section, please include something like the following - the details will vary on exactly how you are using Lookit. 

* Data was collected using Lookit (Scott & Schulz 2017) for participant management, study creation, and data collection. In addition to the automated recruitment of families in the Lookit database, we also [describe your recruiting/advertising approaches.]

* The study design was implemented using Lookit's experiment runner, which uses a JSON object specifying all stimuli and presentation parameters. A text file of the JSON for this experiment can be found at [online location where you upload materials], and the experiment can be previewed by a logged-in user at Lookit at [preview link for your study.]

And the corresponding citation is:
  
* Scott, K., & Schulz, L. (2017). Lookit (part 1): A new online platform for developmental research. Open Mind, 1(1), 4-14. https://doi.org/10.1162/OPMI_a_00002

If you create an experiment that is closely based on one made by another researcher, you should also cite or credit that person's work - reach out to the study creator to learn the best way to do this!

## Our Funders

Finally, a big thank-you to the funders that make Lookit possible:

Halis Family Foundation
The National Science Foundation (1429216, 1823919,  CCF-1231216)
Defense Advanced Research Projects Agency (HR001119S0005)

**Comment by okaycj** [comment]
This has been completed and pushed to production.  Closing issue.
